### PR TITLE
[SPE-16] create header component

### PR DESCRIPTION
**Pull request by m-sekander**
### Type of Change

- [x] New Feature
- [ ] Bug Fix
- [ ] Refactoring
- [ ] Optimization
- [ ] Documentation Update
- [ ] DevOps

### Description

Create responsive header component using Link and NavLink from react-router-dom. Implement the component to App.js. 

Additional adds:
- Design logo
- Change favicon
- Add custom "sc-blue" colour to tailwind config

#### Desktop / Tablet

Desktop Screenshot

#### Large Mobile

Large Mobile Screenshot

#### Small Mobile

Small Mobile Screenshot

### Checklist (delete irrelevant items)
- [x] I have followed coding standards and best practices.
- [x] My changes generate no new warnings or errors.
- [x] I have manually tested my changes.

### CODE REVIEW

File: client/tailwind.config.js [hunk 1]
```diff
@@ -2,7 +2,11 @@
 module.exports = {
   content: ["./src/**/*.{js,jsx,ts,tsx}"],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        "speakers-corner-blue": "#0077c0",
```
Comment (YoussefLaunchUp) [inline client/tailwind.config.js]: nit: would be good to have a shorter name for this. Maybe brand-blue or sc-blue or even just primary if this is the primary color being used in the site

File: client/src/assets/logos/black-logo-transparent.png
Comment (YoussefLaunchUp) [inline client/src/assets/logos/black-logo-transparent.png:1]: great work on the logos

File: client/src/App.tsx [hunk 2]
```diff
@@ -1,27 +1,31 @@
...
+            path="/"
+            element={
+              <div className="flex min-h-[calc(100vh-4.5rem)] items-center justify-center">
+                <h1 className="rounded-xl border bg-gray-200 px-8 py-4 text-lg font-bold text-gray-700 shadow">
+                  Hello World !
+                </h1>
+              </div>
+            }
+          />
+          <Route />
```
Comment (YoussefLaunchUp) [inline client/src/App.tsx]: Lets remove all the routes not being used

**Review by YoussefLaunchUp — CHANGES_REQUESTED** [review]
Great work on the header, some feedback provided.

File: client/src/components/Header/Header.tsx
Comment (YoussefLaunchUp) [inline client/src/components/Header/Header.tsx:1]: This should be called index.tsx so make it easier to import

**Review by YoussefLaunchUp — CHANGES_REQUESTED** [review]

File: client/tailwind.config.js (hunk 1 above)
Comment (m-sekander) [reply client/tailwind.config.js]: shortened it to "sc-blue"

File: client/src/assets/logos/black-logo-transparent.png
Comment (m-sekander) [reply client/src/assets/logos/black-logo-transparent.png:1]: <3

File: client/src/App.tsx (hunk 2 above)
Comment (m-sekander) [reply client/src/App.tsx]: Removed, also added a not found route, for which I created a ticket to make a page for.

File: client/src/components/Header/Header.tsx
Comment (m-sekander) [reply client/src/components/Header/Header.tsx:1]: I usually don't do this as when the tabs are open, it's hard to tell which index.tsx is the index.tsx I want to work on when multiple are open on VScode. However, I made the change so the project can stay consistent.

**Review by YoussefLaunchUp — APPROVED** [review]
Good work

**Pull request merged**